applications/serial_lte_modem/doc/FOTA_AT_commands.rst

```diff
@@ -31,8 +31,9 @@ Syntax
   * ``0`` - Cancel FOTA (during download only).
   * ``1`` - Start FOTA for application update.
   * ``2`` - Start FOTA for modem delta update.
-  * ``7`` - Read modem DFU area size and firmware image offset.
-  * ``9`` - Erase modem DFU area.
+  * ``3`` - Start FOTA for full modem update. Activated when overlay-full_fota.conf is used.
```

Comment:
```suggestion
  * ``3`` - Start FOTA for full modem update.
    Activated when the :file:`overlay-full_fota.conf` configuration file is used.
```
"Activated" makes this sound like it happens automatically. Is that the case?

Reply:
I meant that this feature can be used only when the specific overlay is used. Maybe "Enabled" is better?

Reply:
Best to keep it simple, and just directly say:
```suggestion
  * ``3`` - Start FOTA for full modem update.
    Can only be used when the :file:`overlay-full_fota.conf` configuration file is used.
```

Reply:
Fixed